Resources/Locale/en-US/guidebook/chemistry/effects.ftl

```diff
@@ -267,6 +267,12 @@ reagent-effect-guidebook-make-sentient =
         *[other] make
     } the metabolizer sentient
 
+reagent-effect-guidebook-make-ghost-role =
+    { $chance ->
+        [1] Makes
+        *[other] make
```

Comment:
why one is with uppercase and other is not?

Reply:
Because in the guidebook it would be the start of the sentence.
![image](https://github.com/space-wizards/space-station-14/assets/63975668/99e1cc7c-0c3b-4435-bda3-2e0d6d53118e)